cosmo_tester/framework/test_hosts.py

```diff
@@ -869,26 +870,50 @@ def _set_image_details(self):
             )
 
     def prepare_xfsdump_volume(self):
-        self._logger.info('Preparing XFS volume for XFS-dump')
+        self._logger.info('Preparing XFS dump volume')
         with self.ssh() as fabric_ssh:
             # Identify the still not partitioned volume
-            result = fabric_ssh.run(
-                "lsblk -o NAME,TYPE -ds | awk '$2 == \"disk\" {print $1}'")
-            self._logger.info('>>>> Disk name = `%s`', result.stdout())
+            volume_id = fabric_ssh.run(
+                "lsblk -o NAME,TYPE -ds | awk '$2 == \"disk\" {print $1}'"
+            ).stdout.strip()
+            if not volume_id:
+                self._logger.warning('No XFS dump volume found. '
+                                     'roceeding without.')
```

Comment:
proceeding

Reply:
Switching to reviewing as one block, not by commit- will re-raise comments as required.